app/views/auth/resetPasswordDone.scala.html

```diff
@@ -2,5 +2,5 @@
 
 @views.html.templates.mainAuth(Messages("reset.doneTitle"), tab = "signin") {
   <p>@Messages("reset.done")</p>
-  <p>@Html(Messages("reset.gotoIndex", routes.Application.index))</p>
+  <p>@Html(Messages("reset.gotoIndex", routes.Auth.signIn))</p>
```

Comment:
Why did you changed this? After a "normal" password reset I would imagine to become signed in...

As already mentioned before, we should use different views for password reset and setting up a password for a Pool 1 user...